.cargo/config

```diff
@@ -1,6 +1,10 @@
 [target.'cfg(all(target_arch = "arm", target_os = "none"))']
 runner = "gdb-multiarch -q -x openocd.gdb"
-rustflags = ["-C", "link-arg=-Tlink.x"]
+rustflags = [
+    "-C", "link-arg=-Tlink.x",
+    "-C", "target-cpu=cortex-m7",
+    "-C", "target-feature=+fp-armv8d16",
```

Comment:
Why are we allowed to include amv8 floating point instructions for the Cortex-M7, which is an Armv7 CPU? Is this valid?

Reply:
It would be worth adding comments about why we can use the ARMv8 flag on an ARMv7 processor, since this is actually due to the underlying FPv5 implementation on the STM32H743. https://reviews.llvm.org/D5537

Reply:
Actually, I think what we should actually be doing is making sure that the compiler is getting the right definition of the FPU: `fpv5-dp-d16`. I think this may be a work-around because this isn't getting properly defined.

Reply:
I don't think that exists (anymore).

Reply:
https://code.woboq.org/llvm/llvm/lib/Support/ARMTargetParser.cpp.html#192 indicates you may be right. Mind adding documentation about this here for posterity?